core/src/main/java/org/web3j/protocol/rx/Web3jRx.java

```diff
@@ -167,4 +173,38 @@
      */
     Observable<Transaction> catchUpToLatestAndSubscribeToNewTransactionsObservable(
             DefaultBlockParameter startBlock);
+
+    /**
+     * Creates an observable that emits a notification when a new header is appended to a chain,
+     * including chain reorganizations.
+     *
+     * @return Observable that emits a notification for every new header
+     */
+    Observable<NewHeadsNotification> newHeadsNotifications();
```

Comment:
This is Geth specific right? And some of the other calls?

Reply:
`newHeads` and `logs` are not specific to Geth. Parity supports it as well: https://wiki.parity.io/JSONRPC-Eth-Pub-Sub-Module.html

I'll move other two to the Geth related class